c2corg_ui/static/js/documentediting.js

```diff
@@ -504,6 +504,22 @@ app.DocumentEditingController.prototype.pushToArray = function(object, property,
   app.utils.pushToArray(object, property, value, event);
 };
 
+/**
+ * @param {number} value
+ * @export
+ */
+app.DocumentEditingController.prototype.convertExposureTime = function(value) {
```

Comment:
Please consider moving this function to a dedicated ``imageediting`` controller, as already done for outings.
You could also move the ``date_time`` conversion introduced at https://github.com/c2corg/v6_ui/pull/582/commits/ff1b7f5997ba9fc2aa583d4815f43010d3f78d60